libs/design-system/web-component-card/src/lib/web-component-card.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="image">
+  <img [src]="imageUrl()" />
+</div>
+<div class="cardContent">
+  <h2 class="productTitle" [ngClass]="{ default_style: !webComponentName(), style_change: webComponentName() }">
```

Comment:
Use an `@if` block to toggle between the two behaviors, i.e.
```html
@if (webComponentName()) {
  <span>{{ productTitle() }}</span>
  <h1>{{ webComponentName() }}</h1>
} @else {
  <h1>{{productTitle()}}</h1>
}
```

Then you won't need to use ngClass either